documentation/modules/con-kafka-downgrades-using-cluster-operator.adoc

```diff
@@ -12,8 +12,11 @@ Whether and how the Cluster Operator will perform a downgrade depends on the dif
 * The version of ZooKeeper used by the two Kafka versions
 
 If the downgraded version of Kafka has the same log message format version then downgrading is always possible.
-In this case the Cluster Operator will be able to downgrade by performing a single rolling restart of the brokers.
+In this case, the Cluster Operator performs a downgrade with a single rolling restart of the brokers.
 
-If the downgraded version of Kafka has a different log message format version, then downgrading is only possible if the running cluster has _always_ had `log.message.format.version` set to the version used by the downgraded version. 
+From Kafka {DefaultKafkaVersion}, the `log.message.format.version` is assumed to match the `inter.broker.protocol.version` and doesn't need to be set.
```

Comment:
Again, you need to hardcode the version here and add the `inter.broker.protocol.version` condition.